engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/details/views/content-host-info.html

```diff
@@ -142,7 +142,7 @@ <h4 translate>Content Host Properties</h4>
 
       <div class="detail">
         <span class="info-label" translate>RAM </span>
```

Comment:
@snagoor,
Thanks for PR.
You made changes to display memory value in GB.
It will be good if you add measure unit either in label or in value i.e RAM (GB) or 10 GB.